Doc/whatsnew/3.12.rst

```diff
@@ -126,6 +126,13 @@ Removed
   (and corresponding ``EXPERIMENTAL_ISOLATED_SUBINTERPRETERS``)
   have been removed.
 
+* :mod:`smtpd`, deprecated in Python 3.4.7 and 3.5.4, is now removed according
+  to :pep:`594` schedule. Use aiosmtpd_ PyPI module or any other
```

Comment:
Reword for readability and grammar, and put the removal first as the most important thing. 
```suggestion
* :mod:`smtpd` has been removed according to the schedule in :pep:`594`, 
  having been deprecated in Python 3.4.7 and 3.5.4.
  Use aiosmtpd_ PyPI module or any other
```

Does `:mod:` markup still work if the target doesn't exist? If not we should just use literal ``` `` ``` or no markup at all.

Reply:
Applied.

Sidenote for history: I copied the original wording from the [binhex removal news entry](https://github.com/python/cpython/blame/8150b8cf7fe09d601246f5bdc6456a0e31f31bca/Doc/whatsnew/3.11.rst#L1347).